app/assets/stylesheets/components/gitter_floating.scss

```diff
@@ -30,7 +62,7 @@
 }
 
 @media(max-width: 480px) {
-  .chat-floating-btn {
+  .floating-dropdown {
```

Comment:
Can we move this media query under the `.floating-dropdown ` scope above please?